eclipse-scout-chart/src/chart/Chart.ts

```diff
@@ -72,6 +72,24 @@ export class Chart extends Widget implements ChartModel {
     CENTER: 'center'
   } as const;
 
+  /**
+   * Determines what parts of a charts data is colored with the same colors.
+   */
+  static ColorMode = {
+    /**
+     * Uses one of the other options depending on the chart type.
+     */
+    AUTO: 'auto',
+    /**
+     * Colors each dataset in a different color.
```

Comment:
```suggestion
     * Uses a different color for each dataset.
```
By starting all three description with "Uses" we can help the reader to easily see the differences.